databricks/koalas/frame.py

```diff
@@ -420,6 +422,62 @@ def pipe(self, func, *args, **kwargs):
     def shape(self):
         return len(self), len(self.columns)
 
+    @property
+    def schema(self):
+        return spark.DataFrame.schema.fget(self._sdf)
+
+    @derived_from(spark.DataFrame)
+    def select(self, *cols):
+        return DataFrame(spark.DataFrame.select(self._sdf, *cols))
+
+    @derived_from(spark.DataFrame)
+    def selectExpr(self, *cols):
+        return DataFrame(spark.DataFrame.selectExpr(self._sdf, *cols))
+
+    @derived_from(spark.DataFrame)
+    def describe(self, *cols):
+        return DataFrame(spark.DataFrame.describe(self._sdf, *cols))
+
+    @property
+    def dtypes(self):
+        return spark.DataFrame.dtypes.fget(self._sdf)
+
+    @derived_from(spark.DataFrame)
+    def explain(self, extended=False):
+        return spark.DataFrame.explain(self._sdf, extended)
+
+    @derived_from(spark.DataFrame)
+    def agg(self, *exprs):
+        return DataFrame(spark.DataFrame.agg(self._sdf, *exprs))
+
+    aggregate = agg
+
+    @derived_from(spark.DataFrame)
+    def fillna(self, value, subset=None):
+        return DataFrame(spark.DataFrame.fillna(self._sdf, value, subset))
+
+    @derived_from(spark.DataFrame)
+    def subtract(self, other):
```

Comment:
sry when i mention sub i was referring to subtracting one column from another, i.e. arithmetic operation -.